backup/crypto.go

```diff
@@ -4,12 +4,23 @@ import (
 	"crypto/aes"
 	"crypto/cipher"
 	"crypto/rand"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 )
 
+const (
+	keyLength = 32
+	nonceSize = 12
+)
+
 func encryptFile(source, dest string, key []byte) error {
+
+	if(len(key) != keyLength) {
```

Comment:
Is this backword compatible for those that already have a long key (24 words)?

Reply:
Yes, it is. 

For reference: https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#Generating_the_mnemonic:
- 12 words <=> 128 bit entropy
- 24 words <=> 256 bit entropy

In https://github.com/breez/breezmobile/pull/585/commits/c4a6489c2b24ffb3d349f3a04503a42d84b99d6c#diff-8861906df0c3942376a2de3d76e8e0a597f0e46c0444b24be8e40fb516b94b55R412, for the 12 word phrase, we sha256 128 bits of entropy to get 32 bytes.